extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -57,14 +61,27 @@ export class SubscriptionService {
         this.subscriptionPlatforms_.push(
             new LocalSubscriptionPlatform(
                 this.ampdoc_,
-                subscriptionPlatformConfig.paywallUrl
+                subscriptionPlatformConfig
             )
         );
       });
       resolve();
     });
   }
 
+  /**
+   * This method registers an auto initialized subcription platform with this service.
+   *
+   * @param {string} serviceId
+   * @param {!SubscriptionPlatform} subscriptionPlatform
+   */
+  registerService(serviceId, subscriptionPlatform) {
+    this.subscriptionPlatforms_.push(subscriptionPlatform);
+
+    subscriptionPlatform.getEntitlements()
+        .then(() => this.processEntitlement_());
```

Comment:
Should the `subscriptionPlatform.getEntitlements()` result be passed somewhere?

Reply:
it will be in `this.processEntitlement_` once implemented. It will be passed to `EntitlementStore`